src/transformations/vgridshift.cpp

```diff
@@ -26,7 +26,7 @@ static PJ_XYZ forward_3d(PJ_LPZ lpz, PJ *P) {
     if (P->vgridlist_geoid != nullptr) {
         /* Only try the gridshift if at least one grid is loaded,
          * otherwise just pass the coordinate through unchanged. */
-        point.xyz.z += Q->forward_multiplier * proj_vgrid_value(P, point.lp);
+        point.xyz.z += Q->forward_multiplier * proj_vgrid_value(P, point.lp, Q->forward_multiplier);
```

Comment:
Related to my other comment above, it looks like a mistake here that `Q->forward_multiplier` is used twice. If I understand correctly the multiplier isn't actually applied in `proj_grid_value` but just casually reading this line of code I would expect it to be applied first in `proj_vgrid_value` and then once more in this statement. Shouldn't the multiplier be applied in `proj_vgrid_value`?

Reply:
you're right. that will make things less confusing